app/design/frontend/base/default/template/algoliasearch/internals/configuration.phtml

```diff
@@ -139,9 +139,36 @@ if ($config->isInstantEnabled() && $isSearchPage) {
     </script>';
 }
 
+/** Generate API keys */
+$instantsearchApiKeyParams = $autocompleteApiKeyParams = $config->getAttributesToRetrieve($customerGroupId);
+
+/**
+ * Add custom instantsearch filter restrictions into $instantsearchCustomFilters variable
+ * Docs: https://www.algolia.com/doc/api-client/php/api-keys#generate-key
+ */
+$instantsearchCustomFilters = '';
+
+$instantsearchApiKeyParams['filters'] = $instantsearchCustomFilters.'visibility_search=1';
```

Comment:
I would use facetFilters, the syntax will be clearer, and we let filters empty for people wanting to filter on top of that